tools/junitreport/pkg/parser/gotest/data_parser_test.go

```diff
@@ -50,36 +52,10 @@ func TestExtractTestName(t *testing.T) {
 			testLine:     "=== RUN 1234",
 			expectedName: "1234",
 		},
-		{
-			name:         "basic end",
-			testLine:     "--- PASS: Test (0.10 seconds)",
-			expectedName: "Test",
-		},
-		{
-			name:         "go1.5.1 timing",
-			testLine:     "--- PASS: TestTwo (0.03s)",
-			expectedName: "TestTwo",
-		},
-		{
-			name:         "skip",
-			testLine:     "--- SKIP: Test (0.10 seconds)",
-			expectedName: "Test",
-		},
-		{
-			name:         "fail",
-			testLine:     "--- FAIL: Test (0.10 seconds)",
-			expectedName: "Test",
-		},
-		{
-			name:         "failed print",
-			testLine:     "some other text--- FAIL: Test (0.10 seconds)",
-			expectedName: "Test",
-		},
```

Comment:
Why we're removing these test cases? Looking at the regexes we should be still supporting these.